pkg/test/main/main.go

```diff
@@ -133,13 +137,40 @@ func init() {
 
 	// Enable a muxed cache with partial snapshots
 	flag.BoolVar(&mux, "mux", false, "Enable muxed linear cache for EDS")
+
+	//
+	// These parameters control the the use of the pprof profiler
+	//
+
+	// Enable use of the pprof profiler
+	flag.BoolVar(&pprofEnabled, "pprof", false, "Enable use of the pprof profiler")
+
 }
 
 // main returns code 1 if any of the batches failed to pass all requests
 func main() {
 	flag.Parse()
 	ctx := context.Background()
 
+	if pprofEnabled {
+		// turn on the block profiler
+		log.Println("turn on pprof block profiler")
+		runtime.SetBlockProfileRate(1)
+		if pprof.Lookup("block") == nil {
```

Comment:
Can we use the same pattern here as below where we iterate over the profile types? I would also simplify the output here to just say something like `Turning on block, goroutine and mutex profiling`